machine_translation/sampling.py

```diff
@@ -202,8 +202,8 @@ def _evaluate_model(self):
             trans, costs = \
                 self.beam_search.search(
                     input_values={self.source_sentence: input_},
-                    max_length=3*len(seq), eol_symbol=self.eos_idx,
-                    ignore_first_eol=True)
+                    max_length=3*len(seq), eol_symbol=self.trg_eos_idx,
+                    ignore_first_eol=False)
```

Comment:
Can you change this to `ignore_first_eol=True`, we don't want to change the previous behavior of the example.